content/devices/beryllium.md

```diff
@@ -73,7 +73,7 @@ Please download the belowed needed files and tools:
     sudo systemctl enable enable-ipa sudo systemctl reboot
     ```
 
-Congratulations, if everything went well, now you should be running Droidian.
+Congratulations, if everything went well you should be booting into Droidian.
```

Comment:
The files inside content/devices are now generated by the generate_device_pages.js script, if the changes are present in template files, the changes shoul relect here.

Reply:
i changed this piece of text in templates/* then ran generate_device_pages.js